src/styles/watch.scss

```
@@ -0,0 +1,50 @@
+.watch {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 620px;
+  height: 620px;
+  transform: translateY(10px);
+  &__minute-arrow {
+    position: relative;
+    height: 300px;
+    width: 8px;
+    background-color: blue;
+    transform-origin: bottom;
+    animation-name: watch;
+    animation-duration: 3600s;
+    animation-delay: 1s;
+    animation-iteration-count: infinite;
```

```suggestion
    animation: watch 3600s infinite;
```
Для скорочення коду можна використовувати загальну властивість
`animation-delay` тут взагалі не потрібен

---

Я думаюб що не лишня. Встигає підгрузити деплой. і видно стартову позицію стрілок